.github/workflows/aws-deploy.yml

```diff
@@ -63,6 +63,33 @@ jobs:
           cdk_args: '--require-approval never --context env=dev --progress events --debug'
           actions_comment: false
 
+  cdk-deploy-staging:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/staging' }}
+    needs: tests
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Assume AWS Role
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::878654265857:role/sagebase-github-oidc-sage-ProviderRoleschematicinf-FTTXX1MWO1U0
+          role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
+          role-duration-seconds: 1200
+
+      - name: cdk deploy to org-sagebase-dca-staging (878654265857)
```

Comment:
org-sagebase-dca-staging/org-sagebase-dca-prod

Reply:
@zaro0508 are you suggesting to change the name here to `org-sagebase-dca-staging/org-sagebase-dca-prod`? If I update the name here, should i also update line 114 (see [here](https://github.com/Sage-Bionetworks/schematic-infra/blob/91dfba2e34a5f15fc36d3a8c99742fde4978c112/.github/workflows/aws-deploy.yml#L114))

Reply:
change "org-sagebase-dca-staging" to `org-sagebase-dca-prod` because we are deploying a staging environment to the AWS prod account which is named [org-sagebase-dca-prod](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/organization.yaml#L553).  There is no account named "org-sagebase-dca-staging"

Reply:
Oh I see. Thanks for the explanation @zaro0508